pkg/products/monitoring/dashboards/clusterResources.go

```diff
@@ -29,6 +29,22 @@ func GetMonitoringGrafanaDBClusterResourcesJSON(installationName string) string
 				"titleFormat": "Upgrade",
 				"type": "tags",
 				"useValueForTime": false
+			}, 
+			{
+				"datasource": "Prometheus",
+				"enable": true,
+				"expr": "count by (stage,sku,toSKU)(active_sku{toSKU!=\"\"})",
+				"hide": false,
+				"iconColor": "#FADE2A",
+				"limit": 100,
+				"name": "SKU",
+				"showIn": 0,
+				"step": "",
+				"tagKeys": "stage,sku,toSKU",
+				"tags": "",
+				"titleFormat": "SKU Change (million per day)",
```

Comment:
Why is this saying "(million per day)" that seems like a lot or a misleading title?  I see this is also set in a few other places.

Reply:
The annotation in the grafana dashboard only displays single digits (the same numbers in the sku config). Having "million" per day shows that when the metric is "5" it means "5 million per day".

Reply:
Ok that makes sense, now the "SKU Change" does not seem to describe what it's titling but I am happy to let someone from docs comment on it. This is something that can be changed later if needed.